src/Pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,134 @@
+import { PeopleFilters } from '../../components/PeopleFilters';
+import { Loader } from '../../components/Loader';
+import { useEffect, useState } from 'react';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { PeopleList } from '../../components/PeopleList';
+import { Person } from '../../types';
+
+const addParentReferences = (people: Person[]) => {
+  return people.map(person => {
+    const mother = people.find(human => human.name === person.motherName);
+    const father = people.find(human => human.name === person.fatherName);
+
+    return {
+      ...person,
+      mother,
+      father,
+    };
+  });
+};
+
+const filteredPeople = (visiblesPeople: Person[], query: string) => {
+  return visiblesPeople.filter(
+    person =>
+      person.name.toLowerCase().includes(query.trim().toLowerCase()) ||
+      person.motherName?.toLowerCase().includes(query.trim().toLowerCase()) ||
+      person.fatherName?.toLowerCase().includes(query.trim().toLowerCase()),
+  );
+};
```

Comment:
- Move all these functions to utils/helpers folder/file, `filterVisiblePeople` as well
- `filterPeople` function name should start with a verb